settings/src/components/account-status.js

```diff
@@ -79,27 +86,31 @@ export default function AccountStatus() {
  * @param props.status
  * @param props.headerText
  * @param props.bodyText
+ * @param props.disabled
  */
-function SettingStatusCard( { screen, status, headerText, bodyText } ) {
+function SettingStatusCard( { screen, status, headerText, bodyText, disabled = false } ) {
+	const cardContent = (
+		<CardBody>
+			<StatusIcon status={ status } />
+			<h3 aria-label={ 'Click to enter the ' + headerText + ' setting page.' }>
+				{ headerText }
+			</h3>
+			<p>{ bodyText }</p>
+			<Icon icon={ chevronRight } size={ 26 } className="wporg-2fa__status-card-open" />
+		</CardBody>
+	);
+
+	let classes = 'wporg-2fa__status-card wporg-2fa__status-card-' + screen;
```

Comment:
I'm not sure what our standards are in regards to concatenation, but maybe a template literal is a bit cleaner? 

Not a strong opinion:
```
`wporg-2fa__status-card wporg-2fa__status-card-${screen} `;
```

Reply:
That feels a bit less readable to me, since there's extra symbols cluttering it, and they all run together for me. `prettier` adds spacing around `screen` though, which helps a fair amount:

```js
`wporg-2fa__status-card wporg-2fa__status-card-${ screen } `;
```

The Core standards favor [liberal amounts of spacing](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#spacing). 

I don't feel strongly, but am curious if there's a tangible benefit to template literals, or if it's just personal preference?

Reply:
There are pros and cons for both techniques.... Personal preference :)